### PR TITLE
Issue #72 - Notes for account

### PR DESCRIPTION
**Pull request by javier-gomez-hs**
- Added new attribute "notes" to Account object, Json, Dao, etc.
- Updated SQL files and DDL
- Updated Tests with new attribute

### CODE REVIEW

File: account/src/main/resources/org/killbill/billing/account/ddl.sql [hunk 1]
```diff
@@ -23,6 +23,7 @@ CREATE TABLE accounts (
     country varchar(50) DEFAULT NULL,
     postal_code varchar(16) DEFAULT NULL,
     phone varchar(25) DEFAULT NULL,
+    notes varchar(4096) DEFAULT NULL,
```
Comment (pierre) [inline account/src/main/resources/org/killbill/billing/account/ddl.sql]: Could you also provide a migration file?

See http://docs.killbill.io/0.16/database_migrations.html (see https://github.com/killbill/killbill/blob/work-for-release-0.17.x/invoice/src/main/resources/org/killbill/billing/invoice/migration/V20163917173937__invoice_status_459.sql for example).

Comment (javier-gomez-hs) [inline account/src/main/resources/org/killbill/billing/account/ddl.sql]: Sure @pierre, do I have to create the SQL file manually with the alter table command?
I've never used the **flyway** commands.

Thanks!

Comment (pierre) [inline account/src/main/resources/org/killbill/billing/account/ddl.sql]: > Sure @pierre, do I have to create the SQL file manually with the alter table command?

Yes, that's it. Just make sure to follow the file name pattern from http://docs.killbill.io/0.16/database_migrations.html#_kill_bill.

Comment (javier-gomez-hs) [inline account/src/main/resources/org/killbill/billing/account/ddl.sql]: Thanks @pierre.
I've just added the migration file.

Thanks

**Pull request merged**